### PR TITLE
Add Mailcow deployed_version + Rancher fix

### PR DESCRIPTION
**Pull request by larsl-net**
Add deployed_version for Mailcow, requires Mailcow version 2022-05
Add Regex to deployed_version for Rancher to fix version

### CODE REVIEW

File: content/en/docs/Examples/services.md [hunk 1]
```diff
@@ -513,6 +521,7 @@ service:
         username: <username>
         password: <password>
       json: value
+      regex: v([0-9.]+)
```
Comment (JosephKav) [inline content/en/docs/Examples/services.md:524]: I don't have matomo deployed but does it really use semantic tags and then display it with the leading v in the `deployed_version.url`? I think `v?([0-9.]+)` would work better incase they decide to remove that v

Comment (larsl-net) [reply content/en/docs/Examples/services.md:524]: Matomo returns the version without the v.
I have added the regex to Rancher to remove the v from the Rancher API output.

```json
{
    "baseType": "setting",
    "created": "2022-02-04T15:51:45Z",
    "createdTS": 1643989905000,
    "creatorId": null,
    "customized": false,
    "default": "dev",
    "id": ["server-version"](https://rancher.example.io/v3/settings/server-version),
    "links": {
        "remove": "…/v3/settings/server-version",
        "self": "…/v3/settings/server-version"
    },
    "name": "server-version",
    "source": "env",
    "type": "setting",
    "uuid": "<deleted>",
    "value": "v2.6.3"

}
```

Comment (larsl-net) [reply content/en/docs/Examples/services.md]: Have changed both Points

Comment (JosephKav) [reply content/en/docs/Examples/services.md]: All looks good currently. Thanks for making the PR and those small changes.

Not sure what you mean here though. How is the Rancher API involved in getting the Matomo version? Are you getting the Matomo version from the Rancher API?
We only want to use the API of the service itself to get the version. I think we'd want something like that (showing how to use the rancher API for the `deployed_version` of any service) in a Rancher specific section

**Pull request merged**